src/Orleans.Core/Runtime/GrainReferenceRuntime.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging;
```

Comment:
tried to clean this too.. but my git doesn't think this is a diff...

Reply:
No problems. You can revert the change regardless of what git thinks. Next time